### PR TITLE
All admins can now edit or delete any job

### PR DESCRIPTION
**Pull request by DannyRL**
Changes:
1) Fixed small error where admins could not edit or delete jobs that they did not post. 
2) Rerouted back to job list after creating new job, rather than routing to view-job view. 
3) A dependency or something might have changed with Mongo. I now need to explicitly provide the user and password of the database owner in order for me to connect. Double check that this change still allows everyone else to connect to the DB

Comments:
Login as Test Admin and double check that you can edit or delete jobs I posted. Also hopefully the app still works for everyone else since I made the change in development.js

### CODE REVIEW

**Comment by hxghhhh** [comment]
@czkamin @ssk13

File: modules/users/client/views/admin/view-job.client.view.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-  <div class="pull-right" ng-show="authentication.user._id == job.user._id">
+  <div class="pull-right">
     <a class="btn waves-effect waves-light" ui-sref="jobs.edit({jobId: job._id})">
       <i class="material-icons left">edit</i><span>Edit</span>
     </a>
-    <a class="waves-effect waves-light btn" ng-click="remove();" ng-if="user._id !== authentication.user._id">
+    <a class="waves-effect waves-light btn" ng-click="remove();">
       <i class="material-icons left">delete</i><span>Remove</span>
     </a>
   </div>
```
Comment (ssk13) [inline modules/users/client/views/admin/view-job.client.view.html:12]: same gripe that I had in the other PR - basically, I want this list to look like the 'users' list, but that's not necessarily in the scope of this PR

**Comment by ssk13** [comment]
looks good! Similar complaint to the other one - @hamt3ch I can just restyle those pages after these branches get merged?

**Comment by czkamin** [comment]
nice job. All good from my view - @hamt3ch

File: modules/users/client/views/admin/view-job.client.view.html (hunk 1 above)
Comment (hxghhhh) [inline modules/users/client/views/admin/view-job.client.view.html:12]: Created Task >> PT

**Pull request merged**